cmd/kubelet/app/server.go

```diff
@@ -479,6 +480,7 @@ func SimpleKubelet(client *client.Client,
 	kcfg := KubeletConfig{
 		Address:                   net.ParseIP(address),
 		CAdvisorInterface:         cadvisorInterface,
+	CalculateVolumeDiskUsageFrequency: 5 * time.Minute,
```

Comment:
Yes technically this is a period and not a frequency.  The kubelet flags have the convention of using frequency to mean period.  I am ok with renaming this period if you think it is better.

>
FileCheckFrequency
HTTPCheckFrequency
NodeStatusUpdateFrequency